productService/index.js

```diff
@@ -1,2 +1,3 @@
 module.exports.getProductList = require('./handlers/getProductList');
 module.exports.getProductById = require('./handlers/getProductById');
+module.exports.createProduct = require('./handlers/createProduct');
```

Comment:
Why do you need this file?

Reply:
@TGreyW in `serverless.yml` functions configured from `handler.createProduct` It's mean that function imported from `handle.js` or `handler/index.js`. This file is redundant as i'm correct. Moreover using webpack you can use `es6` modules